CMakeLists.txt

```diff
@@ -485,8 +485,8 @@ if(OPROFILING)
 endif()
 
 if(ENABLE_EVDEV)
-  find_package(Libudev REQUIRED)
-  find_package(Libevdev REQUIRED)
+  find_package(LIBUDEV REQUIRED)
```

Comment:
Not really for this review but just surprised we use `if(LIBUDEV_FOUND...)` when we specify `REQURIED`

Reply:
Like you said, that is out of scope for this PR, but it should probably be fixed later...
```
$ find . -type f -iname '*cmake*' | xargs grep -e LIBUDEV_FOUND -e LIBEVDEV_FOUND
./Externals/libusb/CMakeLists.txt:    if(LIBUDEV_FOUND)
./Externals/hidapi/CMakeLists.txt:  if(CMAKE_SYSTEM_NAME STREQUAL "Linux" AND LIBUDEV_FOUND)
./Source/Core/InputCommon/CMakeLists.txt:if(LIBEVDEV_FOUND AND LIBUDEV_FOUND)
./CMake/FindLIBEVDEV.cmake:#  LIBEVDEV_FOUND - System has libevdev
./CMake/FindLIBUDEV.cmake:#  LIBUDEV_FOUND - System has LIBUDEV
./CMakeLists.txt:  if(LIBUDEV_FOUND AND LIBEVDEV_FOUND)
```